OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -142,7 +141,7 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 		protected DeveloperMode developerMode;
 		protected TechTree techTree;
 
-		public Actor Actor => self;
+		public Actor Actor { get; }
```

Comment:
Fwiw this does feel a bit weird since we use `self` everywhere... but I guess this is unavoidable here.

Reply:
We have a few options for the case where the property and backing field use different names:

1. Turn on the rule and the lost fields names in these few places become acceptable losses for the benefit of having the rule. An alternative point of view is that the rule reduces confusion, because you can't have a field and prop with mismatching names.
2. Turn on the rule, but disable it in places where we want to keep different names. This keeps the names but means future cases would need to know to disable the rule if they also wanted it which might be non-obvious.
3. Don't enable this rule, as we think this cost isn't worht the other benefits.

Reply:
IMO this is fine. Fixing this properly would require rewriting production code, which is something we already want to do, and which is also out of scope of this PR